baseline/client/js/jspsych-countdown.js

```diff
@@ -0,0 +1,54 @@
+jsPsych.plugins["countdown"] = (() => {
+    const plugin = {};
+
+    plugin.info = {
+        name: "countdown",
+        parameters: {
+            duration: {
+                type: jsPsych.plugins.parameterType.INT,
+                default: undefined,
+            },
+            stimulusFormatter: {
+                type: jsPsych.plugins.parameterType.FUNCTION,
+                default: timestamp => `The next part of the experiment will start in ${timestamp}.`,
+            },
+        },
+    };
+
+    plugin.timestamp = ms => {
+        const minutes = Math.max(Math.floor((ms / 1000) / 60), 0);
+        const seconds = Math.max(Math.floor((ms / 1000) % 60), 0);
+        return `${minutes}:${String(seconds).padStart(2, "0")}`;
```

Comment:
In the interests of making this more general-purpose should we check for hours and display them if necessary? And maybe zero-pad the minutes as well?